Backend/ClientActions.php

```diff
@@ -37,20 +37,26 @@ public function __construct(DefaultIntegration $magentoIntegration, APIInterface
      */
     public function getZonesReturnMagentoZone() {
         $magentoDomainName = $this->integrationAPI->getMagentoDomainName();
-        $this->request->setParameters(array('name' => $magentoDomainName));
 
         $response = $this->api->callAPI($this->request);
-
         if($this->api->responseOk($response)) {
-            if(count($response["result"]) === 0) {
-               array_push($response["result"], array(
-                        'name' => $magentoDomainName,
-                        'plan' => array('name' => ''),
-                        'type' => '',
-                        'status' => 'inactive',
-                    )
-                );
+            $magentoZone = null;
+            foreach ($response['result'] as $zone) {
+                if (strstr($magentoDomainName, $zone['name'])) {
+                    $magentoZone = $zone;
```

Comment:
you should check if

domaindomain.com works. A simple check "." character between the str position would be sufficient